db/migrate/20160525214823_promote_plant_parts_to_trait_descriptors.rb

```diff
@@ -0,0 +1,10 @@
+class PromotePlantPartsToTraitDescriptors < ActiveRecord::Migration
```

Comment:
I think you might have misread it - the former one is about _Trait_ Ontology, this one is about _Plant_ Ontology. Two independent things. I did them in a single PR simply because it was convenient implementation-wise.